### PR TITLE
Updated README to fix the correct command

### PR DESCRIPTION
**Pull request by vishnoianil**
for OVN installation and other minor updates

Signed-off-by: Anil Vishnoi <vishnoianil@gmail.com>

### CODE REVIEW

**Comment by vishnoianil** [comment]
@s1061123 PTAL

File: README.md [hunk 1]
```diff
@@ -395,7 +396,7 @@ ansible_ssh_private_key_file=/root/.ssh/dev-server/id_vm_rsa
 **Install Kubernetes**
 
 ```
-ansible-playbook -i inventory/vms.local.generated playbooks/kube-install.yml
+ansible-playbook -i inventory/vms.local.generated playbooks/kube-install-ovn.yml
```
Comment (s1061123) [inline README.md:399]: This `README.md` is general one, so I suppose `kube-install.yml` could be best.
(I hope ovn getting the de-facto standard, however now it isn't...)

**Review by s1061123 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vishnoianil) [reply README.md:399]: This is an example section, that actually specifically says that it's going to install kubernetes with ovn, that's the reason i need to change it for correctness purpose. But if we want to show the general case, we can change this example to general example (probably kubernets + multius + flannel ).?

Comment (s1061123) [reply README.md:399]: Okey, gotcha. it makes sense so let me merge that.

**Pull request merged**